db/migrate/20160106040041_alter_loans.rb

```diff
@@ -0,0 +1,12 @@
+class AlterLoans < ActiveRecord::Migration
+  def change
+    add_column :loans, :name, :string
```

Comment:
This is part of the reason that I try to keep `seeds.rb` exceptionally minimal. If you can open the software and navigate the functionality without the data in `seeds.rb` then it shouldn't be in `seeds.rb`, and overall you *should* be able to do that. There are very few data-level dependencies that should be so deeply embedded that the software can't cope with their absence. Keeping disciplined with what goes into `seeds.rb` makes it easy to know when you should run it because most of the time it shouldn't be changing.